bundles/SyliusResourceBundle/configuration.rst

```diff
@@ -18,7 +18,6 @@ In your ``app/config.yml`` (or in an imported configuration file), you need to d
         resources:
             my_app.entity_key:
                 driver: doctrine/orm
-                manager: default
```

Comment:
I'd just add a scalarNode called "manager" as a required node to \Sylius\Bundle\ResourceBundle\DependencyInjection\Configuration.php

If that's it, I will provide a fix.